### PR TITLE
attendance submission card changes

### PR DESCRIPTION
**Pull request by saadabid2223**

### CODE REVIEW

File: src/components/Utils/getAttendanceStatusColor.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { AttendanceStatusColor } from "src/constant/colors";
+
+
+export const getAttendanceStatusColor = (status) => {
```
Comment (meetwaytech) [inline src/components/Utils/getAttendanceStatusColor.js:4]: change the name of function

File: src/components/Utils/getAttendanceStatusColor.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    switch (status) {
+        case 'present':
```
Comment (meetwaytech) [inline src/components/Utils/getAttendanceStatusColor.js:6]: create const for attendance status

File: src/components/Utils/getAttendanceStatusColor.js (hunk 1 above)
Comment (saadabid2223) [reply src/components/Utils/getAttendanceStatusColor.js]: resolve

File: src/components/Utils/getAttendanceStatusColor.js (hunk 2 above)
Comment (saadabid2223) [reply src/components/Utils/getAttendanceStatusColor.js]: resolve

**Pull request merged**